physionet-django/tests.py

```diff
@@ -0,0 +1,67 @@
+from schema import Query
+from django.test.testcases import TestCase
+import graphene
+import pdb
+
+
+class AnExampleTest(TestCase):
+
+    def test_all_published_projects(self):
```

Comment:
It would be good to have some additional cases that demonstrate that the API is not giving access to content that we would like to keep private.

Reply:
That makes sense, I will add some tests which try to access content not described in the `fields` parameter.